app/controllers/reviewables_controller.rb

```diff
@@ -0,0 +1,164 @@
+class ReviewablesController < ApplicationController
+  requires_login
+
+  PER_PAGE = 10
+
+  before_action :version_required, only: [:update, :perform]
+
+  def index
+    min_score = params[:min_score].nil? ? SiteSetting.min_score_default_visibility : params[:min_score].to_i
+    offset = params[:offset].to_i
+
+    if params[:type].present?
+      unless ['ReviewableUser', 'ReviewableQueuedPost', 'ReviewableFlaggedPost'].include?(params[:type])
```

Comment:
Should we extract that array into a constant so that it's not re-created every time this method is called?

Reply:
discourse-akismet will add a new Class called `ReviewableAkismetPost`. We need a way to discover this dynamically :thinking:

Reply:
I'll add a helper method for this shortly after we merge.